### PR TITLE
Vectorize join hash init.

### PR DESCRIPTION
**Pull request by ienkovich**
Signed-off-by: ienkovich <ilya.enkovich@intel.com>

### CODE REVIEW

File: QueryEngine/JoinHashTable/Runtime/HashJoinRuntime.cpp [hunk 1]
```diff
@@ -34,6 +34,7 @@
 #include "StringDictionary/StringDictionary.h"
 #include "StringDictionary/StringDictionaryProxy.h"
 
+#include <x86intrin.h>
```
Comment (Garra1980) [inline QueryEngine/JoinHashTable/Runtime/HashJoinRuntime.cpp:37]: Is it ok to add this unconditionally?

Comment (Garra1980) [inline QueryEngine/JoinHashTable/Runtime/HashJoinRuntime.cpp:37]: Is it ok to add this unconditionally?

Comment (ienkovich) [reply QueryEngine/JoinHashTable/Runtime/HashJoinRuntime.cpp:37]: It's not unconditional, for non-CUDA only.

Comment (Garra1980) [reply QueryEngine/JoinHashTable/Runtime/HashJoinRuntime.cpp:37]: Right, right, I missed the macro

**Review by Garra1980 — APPROVED** [review]

**Pull request merged**